conf/broker.conf

```diff
@@ -326,6 +326,20 @@ tokenSecretKey=
 # tokenPublicKey=file:///my/public.key
 tokenPublicKey=
 
+### --- SASL Authentication Provider --- ###
+
+# Whether Use SASL Authentication or not.
+isSaslAuthentication=
```

Comment:
This should be enabled through the authentication plugins list instead of an ad hoc flag

Reply:
Thanks, @merlimat @sijie, right, this is a temp change, and will not be exist after the implementation of supporting for web resource. 
As in the current code, this is only used in web resource checking for authentication.
Will add a TODO and link with the issue.